### ISSUE TITLE
 Lizard cloning is wack

### ISSUE THREAD

**Issue by Yogbot-13** [yogstation13/Yogstation#19961]
## Test Merges: 
Not Supplied / None
## Reproduction:
Description: If you clone a lizard, and that clone pops out looking differently like the lizard, showering (among probably other actions) that lizard will cause the lizard to look like their clone.

 Submitted by: yarinoi (718841144351522967)

**Comment by Yarinoi** [comment]
Update: upon further testing, it would seem that clones share enzymes, and if you change the enzymes of one it changes the enzymes of the other. no clue if this is solely a lizard thing or not

**Comment by Yarinoi** [comment]
update: it looks like clones share enzymes. updating the enzymes of one clone will update that of the other. not sure if this is just a lizard thing.

**Comment by ThatGuyPhil** [comment]
I would like to add some possibly important info to this that i have come across:

I just experienced this bug last round (51000, around 1 hours and 20 minutes) while playing as Bime-Jee I randomly changed from his blue color to a light-ish red after a death hallucination. The horns, frills, and belly patterns changed as well. Eye color didn't.
a couple things to note during this were: 
1. it was the same color and phenotype I usually turn into when cloning messes me up (I think)
2. someone made an empty clone of me for genetics reasons which i assume was either scrambled or already pink-ified from cloning.
3. the light-ish red appearence I changed into seems to resemble my first character if you change his species to a lizard. Bime is saved to my second character slot.
Hopefully this helps in getting to the bottom of the cause of the bug and/or prevent other similar bugs from occurring.
